Simplify git / python38-wheel installs

pabelanger opened the pull request:
We don't need to modify our containerfile, we can add these dependencies
into bindep.txt to be installed.

Signed-off-by: Paul Belanger <pabelanger@redhat.com>

AlanCoding commented:
Integration test failures all look related to python requirements

```
FAILED test/integration/test_build.py::test_python_git_requirement[docker] - ...
FAILED test/integration/test_build.py::test_user_python_requirement[podman]
FAILED test/integration/test_build.py::test_python_git_requirement[podman] - ...
```

that could indicate that git is not getting installed or something else.

maybe try rebasing, because a lot of other stuff has been landing.

pabelanger committed:
Rename EE_BUILDER_IMAGE to ANSIBLE_BUILDER_IMAGE

ee-builder downstream is not the same as ee-ansible-builder, so make
this change upstream to avoid confusing people.

Signed-off-by: Paul Belanger <pabelanger@redhat.com>

pabelanger commented:
Pull-request updated, HEAD is now 780ed3a0a0707c41d3f655cb31c075b079e41733

pabelanger committed:
Rename EE_BASE_IMAGE to ANSIBLE_RUNNER_IMAGE

This avoids mixing EE / non EE build args.

Signed-off-by: Paul Belanger <pabelanger@redhat.com>

pabelanger commented:
Pull-request updated, HEAD is now f0acecee46bc006976f4216b6a26b75cc765822d
